test/servico/cst02Test.java

```diff
@@ -55,18 +53,22 @@ public void tearDown() {
     public void testeNotificaçãoDeAusência_DiaInteiro() throws ParseException
     {
         LoginService loginService = new LoginService();
-        assertTrue(loginService.VerificarUsuarioESenha("Funcionario1", "123456"));
+        assertTrue(loginService.verificarUsuarioESenha(
+                "Funcionario1", "123456"));
         
         ProfessorService professorService = new ProfessorService();
-        ProfessorModel professor = professorService.obterProfessorPorNome("Professor1");
+        ProfessorModel professor = professorService.obterProfessorPorNome(
+                "Professor1");
         
         NotificacaoService notificaçãoService = new NotificacaoService();
         List<AusenciaModel> ausencias = notificaçãoService.listarAusencias();        
         assertEquals(0, ausencias.size());
         
-        notificaçãoService.notificarAusencia(professor.id, "25/11/2013", "25/11/2013", "Motivo Declarado", new LinkedList<String>());
+        notificaçãoService.notificarAusencia(professor.id, "25/11/2013",

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The String "25/11/2013" appears 2 times in the file.](https://www.codacy.com/app/calebepb/substituicao/file/2777729600/issues/source?bid=3364462&fileBranchId=3364484#l67)